docs/_articles/en/release-information.md

```diff
@@ -5,6 +5,42 @@ lang: en
 
 ### To update the plugin, please follow the linked  [instructions](./en/getting-started/install/#upgrade-plug-in).
 
+## [v2.9.1](https://github.com/forcedotcom/sfdx-scanner/tree/v2.9.1) (05-21-2021)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.9.0...v2.9.1)
+
+### Release Summary
+
+* RetireJS will now work offlinbe
+* Underlying PMD Engine is upgraded to 6.34.0 (from 6.33.0) 
+* Bug fixes
+
+**Merged pull requests:**
+
+- @W-9296240@ Switch library used to detect binary files [\#411](https://github.com/forcedotcom/sfdx-scanner/pull/411)
+- @W-9282230@ Fixing changelog to update 2.8 release information [\#409](https://github.com/forcedotcom/sfdx-scanner/pull/409)
+- @W-9295472@: Moved vulnerability catalog into different folder, force… [\#408](https://github.com/forcedotcom/sfdx-scanner/pull/408)
+
+## [v2.9.0](https://github.com/forcedotcom/sfdx-scanner/tree/v2.9.0) (05-19-2021)
```

Comment:
If we've pulled 2.9, should we have a section for it in the release notes?

Reply:
ok will merge it.